### PR TITLE
Update PR template for new charts

### LINKED ISSUE #16665: Update PR template for new charts
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by ritazh**
#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #16665 

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [ ] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [ ] Chart Version bumped
- [ ] Variables are documented in the README.md
- [ ] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

**Comment by ritazh** [comment]
cc @prydonius @michelleN

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -24,6 +24,8 @@ will run across your changes and do some initial checks and linting. These check
 very quickly. Please check the results. We would like these checks to pass before we
 even continue reviewing your changes.
 -->
+#### Is this a new chart
+> NOTE: We're experiencing a high volume of PRs to this repo and reviews will be delayed. Please submit your charts to the Helm Hub instead of this repo to make them discoverable to the community. [Here](https://github.com/helm/hub/blob/master/Repositories.md) is how to submit new charts to the Helm Hub.
```
Comment (prydonius) [inline .github/PULL_REQUEST_TEMPLATE.md]: perhaps we should mention something about hosting your own repo: "Please host your own chart repository and submit your repository to the Helm Hub..."?

**Review by prydonius — COMMENTED** [review]
thanks for this @ritazh!

**Review by prydonius — APPROVED** [review]
thanks @ritazh!

/lgtm

**Pull request merged**